api/api.yaml

```diff
@@ -60,6 +60,11 @@ paths:
   /search:
     get:
       description: Free text search over dataset
```

Comment:
Free text search over dataset. If query is empty, this returns all dataset fields, to populate the search drop-down on initial page load.
If query is set, this returns only dataset fields that match the query.